intercept/src/intercept.cpp

```diff
@@ -558,10 +558,10 @@ bool CLIntercept::init()
             "/usr/lib/" CLINTERCEPT_LIBRARY_ARCHITECTURE "/libOpenCL.so.1",
             "/usr/lib/" CLINTERCEPT_LIBRARY_ARCHITECTURE "/libOpenCL.so",
 #endif
-            "/usr/lib/libOpenCL.so.1",
-            "/usr/lib/libOpenCL.so",
-            "/usr/local/lib/libOpenCL.so.1",
-            "/usr/local/lib/libOpenCL.so",
+            "/usr/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so.1",
+            "/usr/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so",
+            "/usr/local/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so.1",
+            "/usr/local/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so",
```

Comment:
Maybe guard these lines after this change?

```suggestion
#ifdef CLINTERCEPT_LIBRARY_DIR
            "/usr/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so.1",
            "/usr/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so",
            "/usr/local/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so.1",
            "/usr/local/" CLINTERCEPT_LIBRARY_DIR "/libOpenCL.so",
#endif
```